catroid/build.gradle

```diff
@@ -26,7 +26,7 @@ buildscript {
         jcenter()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:2.1.3'
+        classpath 'com.android.tools.build:gradle:2.2.3'
```

Comment:
yeah, I'll change that, but what about the issue of excluding files. I have run the task as told by @thmq .